unittests/allplatformstests.py

```diff
@@ -1927,6 +1927,25 @@ def test_options_with_choices_changing(self) -> None:
                 self.assertEqual(item['value'], ['b', 'c'])
                 self.assertEqual(item['choices'], ['b', 'c', 'd'])
 
+    def test_options_listed_in_build_options(self) -> None:
+        """Detect when changed options become listed in build options."""
+        testdir = Path(os.path.join(self.unit_test_dir, '110 list build options'))
+
+        out = self.init(str(testdir))
```

Comment:
There's no purpose in making this a Path object as far as I can tell? Especially since the only time you ever use it, you immediately convert it back to a string?